Makefile

```diff
@@ -0,0 +1,40 @@
+SHELL := /bin/bash
+
+CURRENT_PATH = $(shell pwd)
+REPO = # ghcr.io
+IMAGE = techneatium/techs-checks-latex
+TAG = latest
+FULL_IMAGE = ${REPO}${IMAGE}:${TAG}
+TARGET = F14_CheatSheet.tex
+
```

Comment:
To make the code function on Mac OS (and with podman) the following helper functions had to be added.

```
PODMAN_VM_NAME = podman-vm-pdfbuilder
UNAME := $(shell uname)

.PHONY: macos_podman_preparation
macos_podman_preparation:
# Mount cwd into podman virtual machine on macOS
# Should only be run once
ifeq ($(UNAME), Darwin)
	podman machine init --volume ${CURRENT_PATH}:${CURRENT_PATH} ${PODMAN_VM_NAME}
endif

.PHONY: macos_setup
macos_setup:
# Start the podman virtual machine on macOS
ifeq ($(UNAME), Darwin)
	podman machine start ${PODMAN_VM_NAME}
endif

.PHONY: macos_teardown
macos_teardown:
# Stop the podman virtual machine on macOS
ifeq ($(UNAME), Darwin)
	podman machine stop ${PODMAN_VM_NAME}
endif
```

This was accomplished with help, as I have no real expertise with Docker/Podman.

Reply:
:exploding_head: I have a MacBook Pro that hasn't run MacOS in about ten years. Sometimes I forget how ... interesting running Docker / Podman on BSD can be. :facepalm:

I'll try and find myself a Mac to test with, but at first glance this looks legit.